src/MSBuild.Sdk.SqlProj/MSBuild.Sdk.SqlProj.csproj

```diff
@@ -31,7 +31,7 @@
 
   <Target Name="IncludeDacpacTool" AfterTargets="Build">
     <PropertyGroup>
-      <_DacpacToolSupportedTfms>net6.0;net7.0;net8.0</_DacpacToolSupportedTfms>
```

Comment:
Similar question about `net7.0` support, shouldn't we be supporting this?

Reply:
7.0 is no longer supported